spec/models/trip_spec.rb

```diff
@@ -1,5 +1,88 @@
 require 'rails_helper'
 
 RSpec.describe Trip, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  context '#trips' do
+    it 'has a name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.